types/htmlparser2/htmlparser2-tests.ts

```diff
@@ -1,23 +1,30 @@
 /**
  * Created by staticfunction on 8/4/14.
  */
-import htmlparser = require("htmlparser2");
+import * as htmlparser from 'htmlparser2';
```

Comment:
I think that `import htmlparser = require("htmlparser2");` is the _correct_ thing to do, since it's an Node.js package with `module.exports = ...` in their source

Reply:
You are right. Otherwise it would require the code to be compiled to CommonJS. I'll change.